crates/ruff_python_ast/src/token_kind.rs

```diff
@@ -321,7 +322,6 @@ impl TokenKind {
                 | TokenKind::VbarEqual
                 | TokenKind::CircumFlex
                 | TokenKind::CircumflexEqual
-                | TokenKind::Tilde
```

Comment:
Nit: Can we move this method to the rule that uses it because `TokenKind` is generic and `is_bitwise_or_shift` now no longer returns `true` for all bitwise operators.